docs/migration.md

```
@@ -32,15 +36,22 @@ curl -i https://www.openml.org/api/v1/json/data/1000000
-As per the RFC9457 standard, the "type" field now includes the unique code for the error.
-The "title" field is a human readable summary of the general issue and the "detail" field may provide additional information for the specific request.
-They _will_ be resolvable URIs in the future, providing a page with more information.
+As per the RFC9457 standard, the `type` field now includes the unique code for the error.
+You will notice that the response still contains a `code` of "111" (though as a top level property not embedded in the "error" scope).
+
+The `title` field is a human readable summary of the general issue and the `detail` field may provide additional information for the specific request.
+The URIs provided through `type` _will_ be resolvable in the future, providing a page with more information.
+
+!!! warning
+
+    The`code` field is only included to support the migration of clients, but should be considered **deprecated**. Prefer to use the `type` field to identify the error.
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)